components/console/introduction.rst

```diff
@@ -223,6 +223,13 @@ verbosity levels::
         // ...
     }
 
+.. note::
+
+    For backwards compatibility reasons, these semantic methods do not exist in
+    the ``OutputInterface`` class until Symfony 3.0. They are defined in the
+    different implementations of the interface
+    (e.g. :class:`Symfony\\Component\\Console\\Output\\Output`).
```

Comment:
I'm not sure I understand your comment. I try to say that these methods exist in the different implementations of the interface, for example in the `Output` class.